src/com/yetanalytics/lrs_admin_ui/db.cljs

```diff
@@ -80,6 +80,57 @@
 (s/def ::oidc-auth boolean?)
 (s/def ::oidc-enable-local-admin boolean?)
 
+(s/def ::enable-admin-status boolean?)
+
+(s/def :status.data/statement-count nat-int?)
+(s/def :status.data/actor-count nat-int?)
+(s/def :status.data/last-statement-stored (s/nilable string?))
```

Comment:
This is specifically supposed to be a timestamp string, right?

Reply:
It is, I'll bring over the regex

Reply:
Actually I don't feel it needs to be fully spec'd like that honestly, the xapi-schema regex + validation function is way more than we need here since it is coming from a trusted system. Really it should just check that you can make a `js/Date` out of it without throwing.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/a6a960c907d112abca47959a808c69c32d50c1af